sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/CryptographyAsyncClient.java

```diff
@@ -142,14 +143,13 @@ public class CryptographyAsyncClient {
 
     private final HttpPipeline pipeline;
 
-    private volatile boolean localOperationNotSupported = false;
-    private LocalKeyCryptographyClient localKeyCryptographyClient;
+    private volatile boolean attemptedToInitializeLocalClient;
+    private volatile JsonWebKey jsonWebKey;
```

Comment:
Why do we need this jsonWebKey at the class level? I see this is used only for checking permissions. That should ideally happen inside the local client itself.

Reply:
We can check for the JWK's permissions in the local client.